### ISSUE TITLE
Unable to find type for Gitlab, ProjectSchema in @gitbreaker/browser

### ISSUE THREAD

**Issue by sdc224** [jdalrymple/gitbeaker#798]
Hello,

First of all I want to thanks to this library for easing the gitlab API calls in nodejs. I am creating an electron application and as I am a noob to this technology, I choosed @gitbreaker/browser instead of node (cause electron runs in browser window 😅😅).

While I was using old nodejs gitlab(deprecated now), I was not facing any kind of type issues, nor any missing property(although I was facing some issues, but its ok☺). Now after upgrading it to latest version, I am unable to find ProjectSchema, though it is exported from the library.

I was using a decorator pattern for initializing Gitlab constructor, so I need to have a type for the Gitlab constructor as well. After update, I am unable to do the same, as typescript throwing warnings like "Gitlab is a constructor, but used as a type here". I managed to solve the problem by doing(please mention if I'm wrong)

```js
private gitlab: InstanceType<typeof Gitlab>
```

I am also unable to find the owner property in ProjectSchema, which is really useful in my use case.

Please guide me how can I do the same, and also mention whether I should raise this to stackoverflow community.

Thanks in advance.

**Comment by jdalrymple** [comment]
I add this to the list of things to look into here #793

**Comment by pablobirukov** [comment]
@sdc224 will you please share more code? How do you import from the library? How do you instantiate the API?

**Comment by jdalrymple** [comment]
It was not being exported for the browser export! I fixed this one in #793

**Comment by sdc224** [comment]
```ts
import { PaginatedRequestOptions } from '@gitbeaker/core/dist/types/infrastructure/RequestHelper';
```

For importing ```PaginatedRequestOptions```

Thanks @jdalrymple.
